src/common/config-parser-xml.c

```diff
@@ -743,6 +743,7 @@ static void replace_in_place(char **string, const char *a, const char *b) {
 
 static bool parse_profile(struct parser_data *data, const char *element_name, const XML_Char **attributes) {
     struct profile *p;
+    int channel_count;
```

Comment:
Actually realized when double checking that this is still one thing to change, channel_count should be initialized to -1, otherwise we can compare uninitialized channel_count on line 805. Why -1, so that we can check for 0 on that line and get correct error logs if the failure was earlier in the function.